Cargo.toml

```diff
@@ -17,27 +17,28 @@ hyper-rustls = { version = "0.22.1", optional = true }
 log = "0.4.6"
 pin-project = "1.0.5"
 tokio = { version = "1.2.0", features = ["time"] }
+serde = { version = "1.0", features = ["derive"], optional = true }
```

Comment:
Moving the sse test to the bin directory requires dependencies to be feature enabled.